lib/__tests__/getData.test.js

```diff
@@ -35,11 +35,39 @@ beforeAll(() => {
   fs.readFileSync.mockReturnValue(reportContents)
 })
 
-it('returns parsed config and report contents', () => {
-  const data = getData(configPath, reportPath)
+it('returns parsed config and report contents when no coverage directory is specified in the jest config', () => {
+  const data = getData(configPath)
 
   expect(fs.readFileSync).toHaveBeenCalledTimes(1)
-  expect(fs.readFileSync).toHaveBeenCalledWith(reportPath, 'utf8')
+  expect(fs.readFileSync).toHaveBeenCalledWith(
+    '/workingDir/coverage/coverage-summary.json',
+    'utf8',
+  )
+  expect(data).toStrictEqual({
+    coverages: {
+      branches: { pct: 80 },
+      functions: { pct: 70 },
+      lines: { pct: 60 },
+      statements: { pct: 50 },
+    },
+    thresholds: {
+      branches: 10,
+      functions: 20,
+      lines: 30,
+      statements: 40,
+    },
+  })
+})
+
+it('returns parsed config and report contents when a custom coverage directory is specified in the jest config', () => {
+  require('../jest.config.js').coverageDirectory = 'custom/coverage/directory'
```

Comment:
We can't individually mock the config file in each test since jest.mock calls are hoisted to the top of the file, so just resetting the config to default values at the end of the test 54102f5